scripts/deploy.ts

```diff
@@ -22,15 +24,21 @@ import {
   VerifyingSingletonPaymaster__factory,
 } from "../typechain";
 import { EntryPoint__factory } from "@account-abstraction/contracts";
-import { formatEther } from "ethers/lib/utils";
+import { formatEther, isAddress } from "ethers/lib/utils";
+
+// Deployment Configuration
+const DEPLOYMENT_MODE: "DEV" | "PROD" = "PROD";
```

Comment:
@livingrockrises added this flag to switch b/w dev and prod config

Reply:
can we just take it from .env now?